website/docs/language/state/examples/encryption/openbao.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  encryption {
+    key_provider "openbao" "my_bao" {
+      token = "s.Fg8wA4nDrP08TirpjEXkrTmt"
```

Comment:
:see_no_evil:

Reply:
Make sure you don't length check the token, many places have run into issues with that when upstream switched to the new format. Since we don't have upstream's replication modes, I ended up reverting to the simpler token construction.